deployment/streaming/scripts/batch-process.sh

```diff
@@ -33,18 +34,36 @@ aws emr create-cluster \
 	        \"InstanceCount\": 1,
 	        \"BidPrice\": \"OnDemandPrice\",
 	        \"InstanceGroupType\": \"MASTER\",
-	        \"InstanceType\": \"${BATCH_INSTANCE_TYPE}\",
-	        \"Name\":\"Master\"
+	        \"InstanceType\": \"m4.xlarge\",
+	        \"Name\":\"Master\",
+	        \"EbsConfiguration\": {
+						\"EbsOptimized\": true,
+						\"EbsBlockDeviceConfigs\": [{
+							\"VolumeSpecification\": {
+								\"VolumeType\": \"gp2\",
+								\"SizeInGB\": 1024
+							}
+						}]
+					}
 	      }, {
 	        \"InstanceCount\": ${NUM_EXECUTORS},
 	        \"BidPrice\": \"OnDemandPrice\",
 	        \"InstanceGroupType\": \"CORE\",
-	        \"InstanceType\": \"${BATCH_INSTANCE_TYPE}\",
-	        \"Name\":\"Workers\"
+	        \"InstanceType\": \"r5.xlarge\",
+	        \"Name\":\"Workers\",
+	        \"EbsConfiguration\": {
+						\"EbsOptimized\": true,
+						\"EbsBlockDeviceConfigs\": [{
+							\"VolumeSpecification\": {
+								\"VolumeType\": \"gp2\",
+								\"SizeInGB\": 1024
+							}
+						}]
+					}
 	      }
 	    ]" \
     --scale-down-behavior TERMINATE_AT_TASK_COMPLETION \
-    --auto-terminate \
+		--auto-terminate \
```

Comment:
On further inspection, the whitespace was all kinds of weird all over this file. I made it consistently two space spaces which appears to be at least as consistent as a few different options relative to other files in the project. Kept it as a separate commit so it can be cleanly separated for review.